.github/workflows/coverage-linux.yml

```
@@ -60,6 +60,6 @@ jobs:
       - name: Clean tmp
         run: rm -rf coverage/tmp && rm -rf out
       - name: Upload
-        uses: codecov/codecov-action@v1
+        uses: codecov/codecov-action@81cd2dc8148241f03f5839d295e000b8f761e378
```

```suggestion
        uses: codecov/codecov-action@v3
```

We don't share secrets with this action, do we?